auto/cuew.template.c

```diff
@@ -156,15 +145,87 @@ int cuewInit(void) {
   /* Fetch all function pointers. */
 %LIB_FIND_CUDA%
 
+  result = CUEW_SUCCESS;
+  return result;
+}
 
+static void cuewExitNvrtc(void) {
   if (nvrtc_lib != NULL) {
-%LIB_FIND_NVRTC%
+    /*  Ignore errors. */
+    dynamic_library_close(nvrtc_lib);
+    nvrtc_lib = NULL;
+  }
+}
+
+int cuewNvrtcInit(void) {
```

Comment:
Seems to be private function which is not exposed to a public API. Hence should be using static qualifier.